.github/workflows/deploy.yml

```diff
@@ -128,12 +128,12 @@ jobs:
            --exclude "static/*"
 
       - name: Update Cursor File
-        uses: ../../actions/cursor-deploy
+        uses: ./actions/cursor-deploy
```

Comment:
@pekala Do you remember why we would not do this? Rather annoying that we need to manually update these version numbers.

From here it seems this should work as long as we have a checkout step: https://github.community/t/path-to-action-in-the-same-repository-as-workflow/16952/10

One alternative would be to run a script modifying these files as part of the semantic release.